docs/user/explanation/documentation-structure.rst

```diff
@@ -0,0 +1,43 @@
+How to structure your documentation
+===================================
+
+Structuring your documentation is an important part of the authoring process.
+It's important to think about the reader of the documentation,
+since they are the primary user of documentation.
```

Comment:
I'd go even further here?

We have the `:term:` entry for discoverability that might be handy as well: https://docs.readthedocs.io/en/stable/glossary.html#term-discoverability

```suggestion
A documentation project's ultimate goal is to be read and understood by a reader.
Readers need to be able to :term:`discover <discoverability>` the information that they need and without an intuitive structure,
you risk that readers won't find information that they need or get frustrated on the way.

Having a structure in mind can happen from the very beginning of a documentation project.
Changing and adapting your structure is often also necessary.

.. tip::

   Even if your documentation already has a structure,
   we invite you to keep reading.
   Changing a documentation structure is hard work,
   but the rewards are big.
```

(can write a bullet list of benefits of a good Diátaxis structure as a motivation for people to continue learning about the Framework?)